ansible/roles/motis-proxy/files/Rocket.toml

```diff
@@ -2,9 +2,11 @@
 #
 # SPDX-License-Identifier: AGPL-3.0-or-later
 
+[release]
+ip_header = "X-Forwarded-For"
```

Comment:
Just FYI: RFC 7239 replaced the non-standardised `X-Forwarded-*` headers with the [`X-Forwarded` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Forwarded#transitioning_from_x-forwarded-for_to_forwarded).